components/button.js

```diff
@@ -71,25 +74,28 @@ const buttonSizeClasses = {
  *   Click me!
  * </Button>
  */
-const Button = ({
+export const Button = React.forwardRef(({
   onClick,
   type = "primary",
   size = "md",
   label = "",
   className = "",
+  enabled = true,
   children,
-}) => {
+}, ref) => {
```

Comment:
How does the ref property work? I’m not familiar with it used this way, and I couldn’t find anything relevant in the React documentation.

Reply:
To be honest I don't totally understand, but this is something I found. I found it associated with link buttons?
https://reactjs.org/docs/forwarding-refs.html

We also use it in `NavigationButton` in navigation.js. I imagine I was copying from there too.

Reply:
Oh, dang, I remember now. I completely forgot that I copied that code. Yeah, the ref contains a reference to the button DOM element. It’s puzzling-looking here because we don’t actually see ref used, but the <Link> component uses it internally. So yeah, you have it right.